src/components/TextField/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+interface TextField {
+  readonly name: string
+  readonly label: string
+  readonly placeholder: string
+  readonly inputType: string
+  readonly errorInput: string
+}
+
+const TextField = ({ label, placeholder, inputType, errorInput }: TextField) => {
+  const renderValidInput = () => (
+    <input className="form__input" type={inputType} placeholder={placeholder} />
+  )
+
+  const renderInvalidInput = () => (
+    <>
+      <input className="form__input form__input--invalid" type={inputType} />
+      <div className="form__error">{errorInput}</div>
+    </>
+  )
+
+  return (
+    <div className="form__group">
+      <label className="form__input-label">{label}</label>
+        {errorInput ? renderValidInput() : renderInvalidInput()}
```

Comment:
According to this [comment](https://github.com/Cobda/cobda-web/pull/47/files#r598933928) 

```suggestion
        {renderInput()}
```